examples/multiple-receive/multiple_receive.bal

```diff
@@ -0,0 +1,23 @@
+import ballerina/io;
+import ballerina/lang.runtime;
+
+public function main() {
+    worker w1 {
+        2 -> w3;
+    }
+
+    worker w2 {
+        runtime:sleep(2);
+        3 -> w3;
+    }
+
+    worker w3 returns map<int> {
+        // The worker waits until both values are received.
+        map<int> result = <- {w1, w2};
```

Comment:
Shall we use a record here similar to [multiple-wait](https://ballerina.io/learn/by-example/multiple-wait/)? IMO, A record having two fields to hold the received data is more appropriate and makes the usage more clear.